practices/graded-browser-support.md

```diff
@@ -115,10 +99,9 @@ The following `.browserslistrc` file ([a standard way of sharing target browser
 
 ```nanorc
 defaults
```

Comment:
Add `not ie 11` as `defaults` still includes IE11. [List of browsers without `not ie 11`](https://browserslist.dev/?q=ZGVmYXVsdHMsIGZmID4gNjYsIGNocm9tZSA%2BIDc1LCBzYWZhcmkgPiAxMSwgZWRnZSA%2BIDc4LCBvcGVyYSA%2BIDYx)

Reply:
Ah ok , well spotted 👍 

That site is handy. Browserlist's README says defaults means: `> 0.5%, last 2 versions, Firefox ESR, not dead` and I think at the time I thought ie11 was either `dead` or `< 0.5%`